controller/persistence/migration_v3.go

```diff
@@ -0,0 +1,41 @@
+/*
+	Copyright 2019 Netfoundry, Inc.
```

Comment:
can you make this one 2020 since it's a new file? we need to update all the existing from "2019" to "2019-2020" as well (not on this pr though) :)